README.md

```diff
@@ -211,6 +211,32 @@ Example usage:
   lines so you'll need to be careful to reconstruct this
 - `rails debug:saml['tmp/your_saml_to_debug.txt']`
 
+## Data Loading
+
+There are three types of data that get loaded into this system:
+
+### Database Seeds
+
+These should only be loaded when the application database is initially set up (e.g. for new PR/development deploys or if the staging database needs to be destroyed and recreated). These seeds contain default values for certain tables such as copyrights, licenses, hold sources, and degree types.
+
+The above seed data is loaded automatically during PR builds from Github. During local development it can be loaded during first deployment by running `rake db:seed`.
```

Comment:
`rails db:seed` is the preferred syntax even though it is indeed a `rake` task. I don't recall when they made that change as `rake` used to be the way for sure.